src/main/osd/osd_elements.c

```diff
@@ -793,13 +793,14 @@ static void osdElementHorizonSidebars(osdElementParms_t *element)
 #ifdef USE_RX_LINK_QUALITY_INFO
 static void osdElementLinkQuality(osdElementParms_t *element)
 {
-    // change range to 0-9 (two sig. fig. adds little extra value, also reduces screen estate)
-    uint8_t osdLinkQuality = rxGetLinkQuality() * 10 / LINK_QUALITY_MAX_VALUE;
```

Comment:
This calculation should not just be dropped.

Reply:
Sorry i do not understand , 

its not .. ?  i need to use rxGetLinkQualityPercent  .. as rxGetLinkQuality is  now  0-1023 ?
  
sorry im confused .. what i do wrong?

Reply:
i looked at it again.. 
i cant figure out what is dropped .  😕

Reply:
@mikeller still confused on what was drooped ?

Reply:
```
uint8_t osdLinkQuality = rxGetLinkQuality() * 10 / LINK_QUALITY_MAX_VALUE;
if (osdLinkQuality >= 10) {
    osdLinkQuality = 9;
}
```

Reply:
@mikeller 
NOTE . the other pr for discussion about link quality by rx aux ..
// it has a lot to do with why i chose this switch. and the use of
 rxGetLinkQualityPercent // no need to scale  .. c
 vs
 rxGetLinkQuality native 0-1023 must be scaled ...

`uint16_t rxGetLinkQuality(void)
{
    return linkQuality;
}

uint16_t rxGetLinkQualityPercent(void)
{
    if (!linkQuality) {
        return 0;
    }
    return (linkQualitySource == LQ_SOURCE_RX_PROTOCOL_CRSF) ?  (linkQuality / 3.41) : scaleRange(linkQuality, 0, LINK_QUALITY_MAX_VALUE, 1, 99);
}
`
---
the current logic i used 
// 
`static void osdElementLinkQuality(osdElementParms_t *element)
{
    switch (linkQualitySource) {
        case LQ_SOURCE_RX_PROTOCOL_CRSF:
            tfp_sprintf(element->buff, "%3d", rxGetLinkQualityPercent());
        break;
///..... other case  here in another pr(s) 
        default:
            tfp_sprintf(element->buff, "%1d", rxGetLinkQualityPercent()/10);
        break;
    }
}
`
----  you suggest...
static void osdElementLinkQuality(osdElementParms_t *element)
{
    switch (linkQualitySource) {
        case LQ_SOURCE_RX_PROTOCOL_CRSF:
            tfp_sprintf(element->buff, "%3d", rxGetLinkQualityPercent());
        break;
///..... other case  here in another pr(s) 
        default:
          uint8_t osdLinkQuality = rxGetLinkQuality() * 10 / LINK_QUALITY_MAX_VALUE;
if (osdLinkQuality >= 10) {
    osdLinkQuality = 9;
}
 tfp_sprintf(element->buff, "%1d", osdLinkQuality);
        break;
    }
}
...  the confusion... ?

Reply:
the bold was not to be there..

Reply:
Protoip: Use 3 backticks (\`\`\`) on a line of their own around blocks of code, to make them readable.

Yes, that is what I suggest, unless you can present a good reason for this code to be eliminated.

Reply:
changed to 
```
#ifdef USE_RX_LINK_QUALITY_INFO
static void osdElementLinkQuality(osdElementParms_t *element)
{
    // 0-300 or 1-10
    uint16_t osdLinkQuality = (linkQualitySource == LQ_SOURCE_RX_PROTOCOL_CRSF) ?  (rxGetLinkQuality()  / 3.41) : (rxGetLinkQuality() * 10 / LINK_QUALITY_MAX_VALUE);
    switch (linkQualitySource) {
        case LQ_SOURCE_RX_PROTOCOL_CRSF:
            tfp_sprintf(element->buff, "%3d", osdLinkQuality);
        break;
        default:
            if (osdLinkQuality >= 10) {
                osdLinkQuality = 9;
            }
            tfp_sprintf(element->buff, "%1d", osdLinkQuality);
        break;
    }
}
#endif // USE_RX_LINK_QUALITY_INFO
```
thanks for the pro backtick tip!